3주 차 과제 제출

LEE-sh1673 opened the pull request:
- [X] 예상 대진표
- [X] N개의 최소공배수
- [X] 멀리 뛰기
- [X] 귤 고르기
- [X] 연속 부분 수열 합의 개수
- [X] 괄호 회전하기
- [X] n^2 배열 자르기

LEE-sh1673 committed:
feat: 예상 대진표(12985) 풀이

LEE-sh1673 committed:
feat: N개의 최소공배수(12953) 풀이

LEE-sh1673 committed:
feat: 멀리 뛰기(12914) 풀이

LEE-sh1673 committed:
feat: 귤 고르기(138476) 풀이

LEE-sh1673 committed:
docs: 귤 고르기(138476) 주석 수정

- 문제 링크에 대한 닫는 괄호 누락 수정

LEE-sh1673 committed:
docs: 귤 고르기(138476) 주석 수정

- 코드의 로직을 반영하도록 풀이 설명을 수정

ojs201 commented on SeungHoonLee/3rd week assignment/12953.py:
예전에 승훈님과 유클리드 호제법에 대해 열심히 공부했던 생각이 나는군요.. 그러고 보니 프로그래머스에서 math함수를 지원하지 않았던걸로 기억하는데, 이렇게 하면 되는가보네요?

ojs201 commented on SeungHoonLee/3rd week assignment/12985.py:
매우 간결한 멋진 풀이네요~~

ojs201 commented on SeungHoonLee/3rd week assignment/138476.py:
문제풀이 후 확인해보겠습니다

ojs201 commented on SeungHoonLee/3rd week assignment/12914.py:
문제풀이 후 확인해보겠습니다

LEE-sh1673 committed:
feat: 연속 부분 수열 합의 개수(138476) 풀이

LEE-sh1673 committed:
feat: 괄호 회전하기)76502) 풀이

LEE-sh1673 committed:
feat: n^2 배열 자르기(87390) 풀이

LEE-sh1673 committed:
refactor: 괄호 회전하기(76502) 메서드 명 수정

- 올바른 괄호 문자열 검증 메서드 이름 수정:

as-is:
- is_valid_parens

to-be:
- is_valid_parentheses
